### PR TITLE
Update collection.json for an updated project description

### PR DESCRIPTION
**Pull request by commjoen**

### CODE REVIEW

File: src/data/collection.json [hunk 1]
```diff
@@ -2426,18 +2426,29 @@
...
 			{
 				"name": "download",
 				"url": "https://github.com/commjoen/wrongsecrets"
+			},
+			{
+				"name": "demo",
+				"url": "https://wrongsecrets.herokuapp.com/"
+			},
+			{
+				"name": "K8s demo",
```
Comment (kingthorin) [inline src/data/collection.json:2445]: This is just another ‘demo’

**Comment by kingthorin** [comment]
“container” was probably the collection you were originally after.

https://github.com/OWASP/OWASP-VWAD/blob/master/schema.json

**Comment by commjoen** [comment]
Yes sir, just not sure when, as I first need to fix some things for the project itself.

**Comment by kingthorin** [comment]
Okay no problem, just wanted to make sure it wasn't waiting on something from me/us.

**Comment by kingthorin** [comment]
It's been half a year, please re-open if you plan to address the remaining issues.

**Pull request closed without merging**